index.js

```diff
@@ -682,10 +682,29 @@
   //. > S.maybe(0, R.length, S.Nothing())
   //. 0
   //. ```
-  S.maybe = def('maybe', [a, Function, Maybe], function(x, f, maybe) {
+  var maybe = S.maybe =
+  def('maybe', [a, Function, Maybe], function(x, f, maybe) {
     return fromMaybe(x, maybe.map(f));
   });
 
+  //  catMaybes :: [Maybe a] -> [a]
+  var catMaybes = def('catMaybes', [Accessible], R.chain(maybe([], R.of)));
+
+  //# mapMaybe :: (a -> Maybe b) -> [a] -> [b]
+  //.
+  //. Takes a function and a list, and returns a list containing the value
+  //. of each Just resulting from the application of the function to each
+  //. element of the input list.
+  //.
```

Comment:
How does this sound, @kedashoe?

```javascript
//. Takes a function and a list, applies the function to each element of
//. the list, and returns a list of "successful" results. If the result of
//. applying the function to an element of the list is a Nothing, the result
//. is discarded; if the result is a Just, the Just's value is included in
//. the output list.
```

It's 11:56pm, so I (just) kept my word. :)